HUD/GetWarningScreenMessageTitleHash.md

```diff
@@ -0,0 +1,17 @@
+---
+ns: HUD
+aliases: ["0x81DF9ABA6C83DFF9", "_GET_WARNING_MESSAGE_TITLE_HASH"]
+---
+## GET_WARNING_SCREEN_MESSAGE_HASH
+
+```c
+// 0x81DF9ABA6C83DFF9
+Hash GET_WARNING_SCREEN_MESSAGE_HASH();
+```
+
+```
+NativeDB Introduced: v1290
+```
+
+## Return value
+Returns the warning message title text label hash.
```

Comment:
After some test, I see that returning `0` is resulting when no warning screne is active. Could you add add that please.

Reply:
Sure